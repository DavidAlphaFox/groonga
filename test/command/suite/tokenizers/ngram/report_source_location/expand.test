tokenize \
  'TokenNgram("report_source_location", true)' \
  "ア㌕Ａz" \
  NormalizerAuto
