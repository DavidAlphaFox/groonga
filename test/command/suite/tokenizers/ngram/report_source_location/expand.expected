tokenize   'TokenNgram("report_source_location", true)'   "ア㌕Ａz"   NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "アキ",
      "position": 0,
      "force_prefix": false,
      "source_offset": 0,
      "source_length": 6
    },
    {
      "value": "キロ",
      "position": 1,
      "force_prefix": false,
      "source_offset": 3,
      "source_length": 3
    },
    {
      "value": "ログ",
      "position": 2,
      "force_prefix": false,
      "source_offset": 3,
      "source_length": 3
    },
    {
      "value": "グラ",
      "position": 3,
      "force_prefix": false,
      "source_offset": 3,
      "source_length": 3
    },
    {
      "value": "ラム",
      "position": 4,
      "force_prefix": false,
      "source_offset": 3,
      "source_length": 3
    },
    {
      "value": "ム",
      "position": 5,
      "force_prefix": false,
      "source_offset": 3,
      "source_length": 3
    },
    {
      "value": "az",
      "position": 6,
      "force_prefix": false,
      "source_offset": 6,
      "source_length": 4
    }
  ]
]
